deploy/db/docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: "2.1"
```

Comment:
why do we need this file? If its for ci it should go into .github/workflows I think. Also the db should not be having the db initialized by init.sql since tink now creates all the tables in the first migration.

Reply:
Removed